### DISCUSSION TITLE
How do I get DB's time zone setting in Directus API hook?

### DISCUSSION THREAD

**Discussion by powerseed** [directus/directus#14159] [Archive]
I would like to access DB's time zone in Directus API hook.

It should be the same value as using `SELECT @@global.time_zone;` in Mysql.

![image](https://user-images.githubusercontent.com/42278237/176229968-7de528f4-95e4-49fc-8496-d01ff43b9b0a.png)

I checked the following param mentioned in https://docs.directus.io/extensions/hooks/, but didn't find a field named `time_zone` or anything similar.

![image](https://user-images.githubusercontent.com/42278237/176230113-16507a85-804b-4878-93c7-cb499d7ebcb1.png)

Please help. Thanks!

**Comment by rijkvanzanten** [comment]
Use the `database` context property to run SQL directly to get that number:

```js
const result = await database.raw(`select @@global.time_zone`);
```

**Reply by powerseed** [reply]
Got it. Thanks!
